src/adapter/src/coord/sequencer.rs

```diff
@@ -97,6 +102,17 @@ use super::ReplicaMetadata;
 
 use super::peek::PlannedPeek;
 
+/// Attempts to execute an expression. If an error is returned then the error is returned
+/// to the client and the function is exited.
+macro_rules! try_exec {
```

Comment:
Let me know if you have a better name.

Reply:
Maybe something that makes it obvious that this returns if there was an error: `return_if_err`?

Reply:
That sounds better, changed.